dask_cloudprovider/providers/aws/ecs.py

```diff
@@ -524,6 +519,10 @@ class ECSCluster(SpecCluster):
         Skip cleaning up of stale resources. Useful if you have lots of resources
         and this operation takes a while.
 
+        Default ``False``.
+    use_private_ip: bool (optional)
+        Avoid using a public IP address, e.g. with a private subnet VPC.
```

Comment:
This text could be clearer: 
if `True`, use private PRs, if `False` use public IPs. 
Must be `True` for ec2, but for fargate should be set to `True` when running in a private VPC.

(I don't know if the words above are actually correct)

Reply:
I like your clarification. I'm not certain on the ec2 behavior (only Fargate) so I avoided putting that in the description, but used your phrasing for the rest -- definitely more clear.